<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>activateClass</class>
 <widget class="QMainWindow" name="activateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>activate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="numberlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 127);</string>
    </property>
    <property name="text">
     <string>注册序列号:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);
selection-background-color: rgb(170, 255, 127);
alternate-background-color: rgb(170, 255, 127);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondlineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="thirdLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="forthLineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="thirdLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="activatePushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>激活</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="activate.qrc"/>
 </resources>
 <connections/>
</ui>
